docs/sql-ref-ansi-compliance.md

```diff
@@ -176,14 +273,20 @@ The behavior of some SQL functions can be different under ANSI mode (`spark.sql.
   - `make_date`: This function should fail with an exception if the result date is invalid.
   - `make_timestamp`: This function should fail with an exception if the result timestamp is invalid.
   - `make_interval`:  This function should fail with an exception if the result interval is invalid.
+  - `next_day`: This function throws `IllegalArgumentException` if input is not a valid day of week.
 
 ### SQL Operators
 
 The behavior of some SQL operators can be different under ANSI mode (`spark.sql.ansi.enabled=true`).
   - `array_col[index]`: This operator throws `ArrayIndexOutOfBoundsException` if using invalid indices.
   - `map_col[key]`: This operator throws `NoSuchElementException` if key does not exist in map.
-  - `CAST(string_col AS TIMESTAMP)`: This operator should fail with an exception if the input string can't be parsed.
-  - `CAST(string_col AS DATE)`: This operator should fail with an exception if the input string can't be parsed.
+
```

Comment:
@cloud-fan Odd, why was this removed? It still holds in my DBR 10 build (and it seems right)

Reply:
Seems we used the wrong commits here, the latest file changes for the `sql-ref-ansi-compliance.md` should be:
![image](https://user-images.githubusercontent.com/4833765/135529150-a2ec1660-b3da-4a33-a13f-bc4d81db76d1.png)

Reply:
We decided to allow "group by alias" in ANSI mode. Are you sure it's still forbidden in 10.0? cc @gengliangwang